### PR TITLE
Ayekan mimir deployment

### PR DESCRIPTION
**Pull request by fbegyn**
Deploys Mimir to the ayekan cluster and configures the in-cluster prometheus to start sending data into Mimir.

### CODE REVIEW

File: ayekan/mimir/mimir.yaml
Comment (csilva-cl) [inline ayekan/mimir/mimir.yaml:1]: We should make the s3 credentials a env. variable.

**Review by csilva-cl — CHANGES_REQUESTED** [review]

**Review by cbarria — CHANGES_REQUESTED** [review]
can you please change the commit messages to "(ayekan) commit msg" so we can keep the same notation for each cluster?
also now with the Jira account, branches use the ticket name to be associated with the ticket, let me know, i can explain further 👍🏻

File: ayekan/mimir/mimir.yaml [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+mimir:
+  structuredConfig:
+    common:
+      storage:
+        backend: s3
+        s3:
+          endpoint: "s3.o11y.ls.lsst.org"
+          region: "o11y"
+          access_key_id: "ZE6RTRGFTWG1HV30NMO9"
+          secret_access_key: "VqBygAQpxmTLW48Py5hyC3lpMyZnFP21Kt0mI2xr"
```
Comment (jhoblitt) [inline ayekan/mimir/mimir.yaml:11]: This is a public repo...

File: ayekan/mimir/mimir.yaml [hunk 2]
```diff
@@ -0,0 +1,197 @@
+# Mimir components
+mimir:
+  structuredConfig:
+    common:
+      storage:
+        backend: s3
```
Comment (jhoblitt) [inline ayekan/mimir/mimir.yaml:6]: Looking at the `values.yaml` for the mimir chart, it doesn't appear that there is support for using a k8s secret for the s3 credentials. https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/values.yaml

This is sub-optional as it make it difficult for us to use an external secrets operator.  For the moment, we are going to need to resort to interpolating a value from an env var into the yaml.

**Pull request closed without merging**